pkg/controller/multiarchbuildconfig/multiarchbuildconfig_test.go

```diff
@@ -394,6 +430,13 @@ func TestReconcile(t *testing.T) {
 					},
 				},
 			},
+			wantLog: logOutput(func(e *logrus.Entry) {
```

Comment:
You can just check the log output string, but I don't think that it makes any value in the tests.

Reply:
I want to know how and what the reconciler is going to log because what it has been logging so far wasn't good.
`wantLog` is optional, if you're going to add a new test case and don't want to check any log, just leave it `nil`.

Reply:
I don't see any point. It would be better to check the returned error if a test case fail, instead of testing the actual logging.

Reply:
Well I might revert the tests but we need to improve logs anyway

Reply:
Yea, having better logging is always good, but there is no point in including the logs in the tests.